planb/update_cluster.py

```diff
@@ -330,6 +330,12 @@ def step_forward(ec2: object, volume_id: str, options: dict):
     return True
 
 
+def ssh_command_works(odd_host: str):
```

Comment:
This does not feel good with the boolean return value. Current name is not so bad (only the '?' is missing, but not possible). Shorter would be `is_sshable` or `is_ssh_connectable`.